.lycheeignore

```diff
@@ -1,2 +1,4 @@
 file://.*
```

Comment:
Are you sure you want to keep skipping file links? It may be that some internal links are relevant?